Source/Blazorise/Styles/vendor/_floating-ui.scss

```diff
@@ -0,0 +1,13 @@
+.b-floating {
```

Comment:
TBH, I don't like this `floating` name. Can we find an alternative?

- `position-strategy`, or
- `dropdown-position-strategy`, or
- `dropdown-menu-position-strategy`

---

I would go even further. Create new classes for each of CSS providers so that names match their convention.

AntDesign:  `and-dropdown-menu-position-strategy and-dropdown-menu-position-strategy-{type here}`
Bootstrap:  `dropdown-menu-position-strategy dropdown-menu-position-strategy-{type here}`
Tailwind:  `max-w-max top-0 left-0 fixed`, or `max-w-max top-0 left-0 absolute`

Reply:
I guess I wanted it to be representative of the feature so you can easily associate it with the floating-ui lib. I don't mind the suggestions, will go ahead with the suggested changes.

Reply:
Yeah, but we also want to abstract it away from the library.